src/content/docs/licenses/license-information/usage-plans/new-relic-one-usage-plan-descriptions.mdx

```diff
@@ -283,9 +283,11 @@ The following List Price tables identify per-unit prices. For more information,
         </tbody>
       </table>
 
-#### Add-ons 
+The List Price table below includes pricing details for several add-ons, including: 
```

Comment:
@zuluecho9 I don't see a table title "List Price" so how about just rejiggering the sentence like this: 

The table below shows list price details for several add-ons, including: